cdap-ui/app/cdap/components/Overview/index.js

```diff
@@ -81,16 +127,75 @@ export default class Overview extends Component {
       paginationContainer.scrollTop -= 120;
     }
   }
-  componentDidMount() {
-    this.scrollEntityToView();
+  bindKeyboardShortcuts() {
+    this.mousetrap = new Mousetrap(document);
```

Comment:
Don't think so. Ran a profile check on google chrome to see if there are any leaks but found none. Nevertheless have added a `reset` call + deletes the object before creating a new one.